ignite/handlers/checkpoint.py

```diff
@@ -354,7 +358,13 @@ def __call__(self, engine: Engine) -> None:
                 global_step = engine.state.get_event_attrib_value(Events.ITERATION_COMPLETED)
             priority = global_step
 
-        if self._check_lt_n_saved() or self._saved[0].priority < priority:
+        def _compare_fn(old: Union[int, float], new: Union[int, float]) -> bool:
```

Comment:
```suggestion
        def _compare_fn(self, old: Union[int, float], new: Union[int, float]) -> bool:
```

Reply:
I see your point.
Updated in the latest commit.

Thanks.

Reply:
Thanks! At first I was thinking that the method could be static, then realized that it has `self.greator_or_equal`, so I think it could be interesting to fetch `self._saved[0].priority` inside this function...